test/server/cards/02_SHD/leaders/FennecShandHonoringTheDeal.spec.ts

```diff
@@ -99,5 +99,29 @@ describe('Fennec Shand, Honoring the Deal', function () {
                 expect(context.battlefieldMarine.location).toBe('discard');
             });
         });
+
+        describe('Fennec Shand\'s deployed ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        leader: { card: 'fennec-shand#honoring-the-deal', deployed: true },
+                        hand: ['reinforcement-walker'],
+                        resources: 8
+                    },
+                    player2: { }
+                });
+            });
+
+            it('should not allow the player to play a unit with cost 4 or less when he does not have any targetable units in hand', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.fennecShand);
```

Comment:
Could we simplify / clarify this test a little bit by setting her as exhausted to start (can just `exhausted: true` in setup) and then do `expect(context.fennecShand).not.toHaveAvailableActionWhenClickedBy(context.player1);`